src/App.js

```diff
@@ -1,9 +1,15 @@
+import React, { Component } from 'react';
```

Comment:
Please try to compile with no warnings.

![image](https://user-images.githubusercontent.com/52723192/103827369-df2de280-5089-11eb-9397-c024635b1b99.png)